drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/SalienceSynchronizer.java

```diff
@@ -73,12 +71,9 @@ public void updateSalienceColumnValues( final AttributeCol52 modelColumn ) {
     private void setSalienceByRowNumbers( final AttributeCol52 modelColumn,
                                           final int iModelColumn ) {
         //If Salience values are reverse order derive them and update column
-        int salience = ( modelColumn.isReverseOrder() ? model.getData()
-                .size() : 1 );
-        for ( int rowNumber = 0; rowNumber < model.getData()
-                .size(); rowNumber++ ) {
-            final List<DTCellValue52> modelRow = model.getData()
-                    .get( rowNumber );
+        int salience = (modelColumn.isReverseOrder() ? model.getData().size() : 1);
```

Comment:
Do we really need this parentheses?

Reply:
we do not, will remove in a separate commit, thank you

Reply:
thanks